zulipterminal/ui_tools/boxes.py

```diff
@@ -490,6 +491,19 @@ def autocomplete_emojis(self, text: str, prefix_string: str
 
         return emoji_typeahead, emojis
 
+    def footer_notify_invalid_stream(self) -> None:
+        invalid_stream_error = (
+            'Invalid stream name. Use {} or {} to autocomplete.'.format(
+                primary_key_for_command('AUTOCOMPLETE'),
+                primary_key_for_command('AUTOCOMPLETE_REVERSE'))
```

Comment:
It might be more vivid to style the commands in `'code'`, similar to the way we show other commands in hints in the footer.